index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main-doc">
+      <nav id="navbar">
+        <header>
+          <h1>JS Documentation</h1>
+          <a class="nav-link" href="#Introduction">Introduction</a>
+          <a class="nav-link" href="#What_you_should_already_know">What you should already know</a>
+          <a class="nav-link" href="#JavaScript_and_Java">JavaScript and Java</a>
+          <a class="nav-link" href="#Function_declarations">Function declarations</a>
+          <a class="nav-link" href="#Reference">Reference</a>
+        </header>
+      </nav>
+      <section class="main-section" id="Introduction">
+        <header>
+          <h1>Introduction</h1>
+          <p>
+            <code>
+              Lorem ipsum dolor sit amet.
+            </code>
+          </p>
+          <p>
+            <li>
+
+            </li>
+          </p>
```

Comment:
Kindly use the correct semantic rules here. 
`li` tag should not be inside the `p` tag. 
Remove the tags and use them separately.

Reply:
- [ ] You can consider using suggestions instead of orders so your comment is phrased politely, e.g `Kindly consider separating the tags so your HTML is semantic`